etc/templates/config/generic/syscheck.agent.template

```diff
@@ -26,6 +26,8 @@
     <ignore>/etc/svc/volatile</ignore>
     <ignore>/sys/kernel/security</ignore>
     <ignore>/sys/kernel/debug</ignore>
+    <ignore>/dev/core</ignore>
+    <ignore>/proc</ignore>
```

Comment:
`<ignore>/proc</ignore>` will ignore directories containing `"/process"`, better use `sregex` and `^/proc`.